src/store/peer/actions.ts

```diff
@@ -1,21 +1,34 @@
 import { ActionType, createAction, createAsyncAction } from 'typesafe-actions';
-import { Peer } from './types';
-import {ApiError, DeleteResponse, RequestPayload} from '../../services/api-client/types';
+import {Peer, PeerGroupsToSave} from './types';
+import {ApiError, ChangeResponse, DeleteResponse, RequestPayload} from '../../services/api-client/types';
+import {Group} from "../group/types";
```

Comment:
Codacy has a fix for the issue: [Replace `Group` with `·Group·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import { Group } from "../group/types";
```